openvdb/tree/Tree.h

```diff
@@ -113,6 +113,10 @@ class OPENVDB_API TreeBase
     /// is given as element NodeType::LEVEL in the return vector. Thus, the size
     /// of this vector corresponds to the height (or depth) of this tree.
     virtual std::vector<Index32> nodeCount() const = 0;
+    /// Return a vector with active tile counts. The number of active tiles at level NodeType
+    /// is given as element NodeType::LEVEL in the return vector. Thus, the size
+    /// of this vector corresponds to the height (or depth) of this tree.
```

Comment:
This should include a discussion of why this is different than the nodeCount() above.

"active tile counts" could be read ambiguously, sometimes people use that to refer to active tiles without children (ie, active constant regions) but not active tiles with children (ie, active nodes).

Suggested text.
Return a vector with active tile counts.   While nodeCounts returns the number of allocated nodes at each level, regions of the tree may have data but not have nodes because they have activated constant regions.  This returns both the count of both of these types of tiles.